modules/data.atmosphere/R/download.raw.met.module.R

```diff
@@ -38,7 +38,7 @@
     
   } else if (register$scale == "site") {
     # Site-level met
-    raw.id <- PEcAn.utils::convert.input(input.id = NA,
+    raw.id <- convert.input(input.id = NA,  ### PEcAn.utils::
```

Comment:
Should not be removing package names from functions

Reply:
Ah, whoops, that was for debugging.  I can fix that.